docs/flocker-features/pure-storage-configuration.rst

```diff
@@ -0,0 +1,13 @@
+.. _pure-storage-backend:
+
+=================================
+Pure Storage Block Device Backend
+=================================
+
+.. begin-body
+
+Pure Storage provides a Flocker backend plugin.
+
+For more information, including installation, testing and usage instructions, visit the `Pure Storage Flocker Driver on GitHub <https://github.com/sdodsley/purestorage-flocker-driver>`_
```

Comment:
I ninja edited the README, its just pointing to the latest docs page so the link wont go stale.

I'll upgrade one of my test envs to 1.13 and make sure things are still ok. Were there any major changes I should check for specifically?